lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -65,20 +38,55 @@ def to_representation(self, course_overview):
             "start_type": start_type,
             "end": course_overview.end,
             "course_image": course_overview.course_image_url,
+            "subscription_id": course_overview.clean_id(padding_char='_'),
+
+            # Note: The following 2 should be deprecated.
             "social_urls": {
                 "facebook": course_overview.facebook_url,
             },
             "latest_updates": {
                 "video": None
             },
-            "video_outline": video_outline_url,
-            "course_updates": course_updates_url,
-            "course_handouts": course_handouts_url,
-            "discussion_url": discussion_url,
-            "subscription_id": course_overview.clean_id(padding_char='_'),
-            "courseware_access": has_access(request.user, 'load_mobile', course_overview).to_json() if request else None
+
         }
 
+        request = self.context.get('request', None)
+        if request:
+            response_data['courseware_access'] = has_access(
+                request.user,
+                'load_mobile',
+                course_overview
+            ).to_json()
+
+            response_data['course_about'] = reverse(
+                'about_course',
+                kwargs={'course_id': course_id},
+                request=request,
+            )
+            response_data['course_updates'] = reverse(
+                'course-updates-list',
+                kwargs={'course_id': course_id},
+                request=request,
+            )
+            response_data['course_handouts'] = reverse(
+                'course-handouts-list',
+                kwargs={'course_id': course_id},
+                request=request,
+            )
+            if course_overview.is_discussion_tab_enabled():
+                response_data['discussion_url'] = reverse(
+                    'discussion_course',
+                    kwargs={'course_id': course_id},
+                    request=request,
+                )
+            response_data['video_outline'] = reverse(
+                'video-summary-list',
+                kwargs={'course_id': course_id},
+                request=request,
+            )
```

Comment:
`reverse` works with or without a request object.  It just provides the server name.  It would probably be useful to return this info even if a request is not provided.  

    with request: http://testserver/api/courses/v1/courses/{course_id}/about
    without request: /api/courses/v1/courses/{course_id}/about